docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -0,0 +1,280 @@
+# Integrate OpenVINO™ with Your Application {#openvino_docs_Integrate_OV_with_your_application}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   openvino_docs_OV_Runtime_UG_Model_Representation
+   openvino_docs_OV_Runtime_UG_Infer_request
+
+@endsphinxdirective
+
+The following diagram illustrates the typical application development workflow:
```

Comment:
I think we can remove this diagram and this text. We need to start with `Use OpenVINO™ Runtime API to Implement Inference Pipeline` which can be renamed to `Introduction`